src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/CaseNoteControllerTest.kt

```diff
@@ -109,6 +113,7 @@ class CaseNoteControllerTest {
     fun `can get an individual case note`() {
       val id = UUID.randomUUID()
       val caseNote = caseNoteFactory.create(id = id, subject = "subject", body = "body")
+      whenever(authUserRepository.save(any())).thenReturn(authUserFactory.create())
```

Comment:
This is probably safer to re-use if we pass through the original argument rather than creating a new object:
```kt
whenever(authUserRepository.save(any())).then(AdditionalAnswers.returnsFirstArg<AuthUser>())
```
This way, we don't mutate the object that is potentially set up by the test -- and we can use this pattern instead of `.thenReturn(user)`.